### PR TITLE
DEP: Remove coordinate transformations moved to pysatMadrigal

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses #167. (partial)

The functions used for JRO have been moved to pysatMadrigal.

## Type of change

- Breaking change (fix or feature that would cause existing functionality
      to not work as expected)

# How Has This Been Tested?

with pytest

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

**Comment by aburrell** [comment]
The files you changed look good, but coverage dropped related to the files under consideration.  I went in and updated the relevant unit tests.

File: pysat/tests/test_utils_coords.py [hunk 1]
```diff
@@ -60,379 +36,83 @@ def test_adjust_cyclic_data_custom(self):
         assert test_angles.max() < 180.0
         assert test_angles.min() >= -180.0
 
+
+class TestLonSLT():
+    def setup(self):
+        """Runs after every method to clean up previous testing."""
+        self.test_inst = None
```
Comment (rstoneback) [inline pysat/tests/test_utils_coords.py:43]: is it worth moving away from testInst?

Comment (jklenzing) [reply pysat/tests/test_utils_coords.py:43]: We might want to move away from using test as part of the variable names in general.  I don't think this causes any specific issues here, but functions that begin with "test" will be collected as part of the unit tests regardless of where they are in the code. I'm kind of leaning toward reserving "test" as a keyword for functions and classes that are meant to be run as tests.

Comment (aburrell) [reply pysat/tests/test_utils_coords.py]: I changed it to `py_inst` after discussion with @jklenzing

**Pull request merged**